src/app/smoothie/conference/conference.component.jsx

```diff
@@ -1,22 +1,72 @@
 import React from 'react';
 import PropType from 'prop-types';
 import './conference.css';
+import Modal from './conferenceModal.component';
 
-const Conference = ({ name, timeBegin, timeEnd }) => (
-  <div className="columns">
-    <div className="column">
-      <div className="conference">
-        <div className="conference-opt"><a><i className="fa fa-arrows-h circle" /></a><span className="conference-time">{`${timeBegin}h00 > ${timeEnd}h00`}</span><a><i className="fa fa-lock circle" /></a></div>
-        <div className="conference-title">{name !== undefined ? name : 'Temps libre'}</div>
+class Conference extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalVisible: false,
+    };
+
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal() {
+    this.setState((prev) => {
+      const newState = !prev.isModalVisible;
+      return {
+        isModalVisible: newState,
+      };
+    });
+  }
+
+  render() {
+    return (
```

Comment:
tu peux destructurer ici 👨‍🚀 
const {name, timeBegin, timeEnd, autres props...} = this.props